### PR TITLE
Badge on icon

### PR DESCRIPTION
**Pull request by GuptaEesh**
Badge-on-icon contains:
>Types

1. Based on image size
2. Based on special material icons with a class "mat"

badgeOnIcon

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
LGTM

File: components/badge-on-icon.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    min-width: 1.2em;
+    padding: 5px;
+    position: absolute;
+    right: 0;
+    text-align: center;
+    top: -2px;
+}
+.mat{
+    top:-10px;
+    right:-5px;
```
Comment (Hamdrive) [inline components/badge-on-icon.css:29]: Prefer to not use negative values here, go for better alternatives

**Review by Hamdrive — COMMENTED** [review]
Changes suggested

File: components/badge-on-icon.css (hunk 1 above)
Comment (GuptaEesh) [reply components/badge-on-icon.css]: Used **transform: translate()** instead of top and right as suggested by you previously.

**Pull request merged**